### ISSUE TITLE
Latest release `QML v0.7.1` crashes during self test on `Julia 1.9.0-rc1`

### ISSUE THREAD

**Issue by cmey** [JuliaGraphics/QML.jl#160]
```
julia> versioninfo()
Julia Version 1.9.0-rc1
Commit 3b2e0d8fbc1 (2023-03-07 07:51 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin21.4.0)
  CPU: 16 × Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, skylake)
  Threads: 1 on 16 virtual cores



(QML-work) pkg> st
Status `~/Code/julia/QML-work/Project.toml`
  [2db162a6] QML v0.7.1


 
(QML-work) pkg> test QML

[...]

     Testing Running tests...
Running tests from badqml.jl
Assertion failed: (jl_datatype_nfields(dt) == 1), function boxed_cpp_pointer, file /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/usr/local/include/jlcxx/type_conversion.hpp, line 641.

[97737] signal (6): Abort trap: 6
in expression starting at /Users/christophemeyer/.julia/packages/QML/QJmAc/test/badqml.jl:7
__pthread_kill at /usr/lib/system/libsystem_kernel.dylib (unknown line)
pthread_kill at /usr/lib/system/libsystem_pthread.dylib (unknown line)
abort at /usr/lib/system/libsystem_c.dylib (unknown line)
__assert_rtn at /usr/lib/system/libsystem_c.dylib (unknown line)
_ZN5jlcxx17boxed_cpp_pointerI7QStringEENS_10BoxedValueIT_EEPS3_P14_jl_datatype_tb at /Users/christophemeyer/.julia/artifacts/e8386fae7586d09d5c4cffc8634174b499dc7bac/lib/libjlqml.dylib (unknown line)
_ZN5jlcxx6detail11CallFunctorI7QStringJPKtiEE5applyEPKvNS_13WrappedCppPtrEi at /Users/christophemeyer/.julia/artifacts/e8386fae7586d09d5c4cffc8634174b499dc7bac/lib/libjlqml.dylib (unknown line)
fromUtf16 at /Users/christophemeyer/.julia/packages/CxxWrap/IdOJa/src/CxxWrap.jl:618
QString at /Users/christophemeyer/.julia/packages/QML/QJmAc/src/QML.jl:105 [inlined]
load_qml at /Users/christophemeyer/.julia/packages/QML/QJmAc/src/QML.jl:43
#loadqml#1 at /Users/christophemeyer/.julia/packages/QML/QJmAc/src/QML.jl:62
loadqml at /Users/christophemeyer/.julia/packages/QML/QJmAc/src/QML.jl:56
unknown function (ip: 0x10b9f936c)
ijl_apply_generic at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
do_call at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
eval_body at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
eval_body at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
jl_interpret_toplevel_thunk at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
jl_toplevel_eval_flex at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
jl_toplevel_eval_flex at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
ijl_toplevel_eval_in at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
eval at ./boot.jl:370 [inlined]
include_string at ./loading.jl:1858
ijl_apply_generic at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
_include at ./loading.jl:1918
include at ./client.jl:478
unknown function (ip: 0x10b9a0062)
ijl_apply_generic at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
macro expansion at /Users/christophemeyer/.julia/packages/QML/QJmAc/test/runtests.jl:14 [inlined]
macro expansion at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-macmini-x64-5.0/build/default-macmini-x64-5-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/Test/src/Test.jl:1586 [inlined]
macro expansion at /Users/christophemeyer/.julia/packages/QML/QJmAc/test/runtests.jl:12 [inlined]
macro expansion at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-macmini-x64-5.0/build/default-macmini-x64-5-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/Test/src/Test.jl:1498 [inlined]
top-level scope at /Users/christophemeyer/.julia/packages/QML/QJmAc/test/runtests.jl:12
jl_toplevel_eval_flex at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
jl_toplevel_eval_flex at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
ijl_toplevel_eval_in at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
eval at ./boot.jl:370 [inlined]
include_string at ./loading.jl:1858
ijl_apply_generic at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
_include at ./loading.jl:1918
include at ./client.jl:478
unknown function (ip: 0x10b9a0062)
ijl_apply_generic at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
do_call at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
eval_body at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
jl_interpret_toplevel_thunk at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
jl_toplevel_eval_flex at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
jl_toplevel_eval_flex at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
ijl_toplevel_eval_in at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
eval at ./boot.jl:370 [inlined]
exec_options at ./client.jl:280
_start at ./client.jl:522
jfptr__start_30711.clone_1 at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/sys.dylib (unknown line)
ijl_apply_generic at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
true_main at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
jl_repl_entrypoint at /Users/christophemeyer/.julia/juliaup/julia-1.9.0-rc1+0.x64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
Allocations: 5595660 (Pool: 5590041; Big: 5619); GC: 8
ERROR: Package QML errored during testing (received signal: 6)
```

**Comment by ufechner7** [comment]
Perhaps this can be closed now that 0.8 is released?

**Comment by cmey** [comment]
Yes, QML v0.8.0 on Julia 1.9.3 no longer crashes with this error.
